docs/source/miscellaneous_environment_variables.rst

```diff
@@ -0,0 +1,15 @@
+.. _miscellaneous_environment_variables:
+
+Miscellaneous Environment Variables
+===================================
+.. list-table::
+  :header-rows: 1
+
+  * - Variable
+    - Description
+  * - ``TORCH_FORCE_WEIGHTS_ONLY_LOAD``
+    - If set to [``1``, ``y``, ``yes``, ``true``], the torch.load will use ``weight_only=True``. For more documentation on this, see :func:`torch.load`.
+  * - ``TORCH_CPP_LOG_LEVEL``
```

Comment:
should live next to TORCH_LOG and SHOW_CPP_STACKTRACES I think

Reply:
good call, will move